_ingest-pipelines/processors/date-index-name.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: default
+title: Date index name
+parent: Ingest processors
+nav_order: 55
+---
+
+# Date index name
+
+The `date_index_name` processor is used to point documents to the correct index based on the date or timestamp field within the document. The processor can be used for purposes such as log retention and index partitioning.
+
+The following is the syntax for the `date_index_name` processor:
+
+```json
+{
+  "date_index_name": {
+    "field": "date_field or timestamp_field",
+    "date_rounding": "M"
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `date_index_name` processor.
+
+Parameter | Required | Description |
+|-----------|-----------|-----------|
+`field`  | Required  | The date or timestamp field in the incoming document. <Supports [template snippets]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/#template-snippets).> |
```

Comment:
Missing parameter? `index_name_prefix`?

Reply:
Is this parameter an option? Here's a screenshot of the autopopulate options when using the Dev Tools console.
<img width="694" alt="Screenshot 2023-11-07 at 5 11 33 PM" src="https://github.com/opensearch-project/documentation-website/assets/105296784/332f76c0-ee40-4b37-a3e3-a7e4b183bba9">

Reply:
It is an optional parameter.